File: fizz.f90
program fizz
  do 100 i = 0, n
    if (mod(i, 15) .EQ. 0) then
      print * , "FizzBuzz"
    else if (mod(i, 5) .EQ. 0) then
      print *, "Buzz"
    else if (mod(i, 3) .EQ. 0) then
      print *, "Fizz"
    else
      print *, i
    end if
  enddo
end program fizz

File: Fizz.scala
object Fizz {
  def main(args: Array[String]) {
    for(i <- 0 to 99) {
      if (i % 15 == 0) {
        println("FizzBuzz")
      } else if (i % 5 == 0) {
        println("Buzz")
      } else if (i % 3 == 0) {
        println("Fizz")
      } else {
        println(i)
      }
    }
  }
}
